### ISSUE TITLE
Latest build fails to build dependencies 

### ISSUE THREAD

**Issue by EthanLatimerGB** [sufst/back-end#38]
### Description

Description of the bug. 

### Steps To Reproduce

1. Followed all steps up to step 3 in SUFST documentation under "Installing the Back-End Server"
2. Completed step `pip install wheel pipenv`
3. Entered the command `pipenv install`

#### Failed on following Distributions

Ubuntu 22.04 LTS - Fresh install on a VM. 
Fedora Server 36 - Fresh install on a VM.
Manjaro Linux (Latest 21-10-22 packages).

Intermediate server builds with no issues.

### Expected and Received Behaviour

Expected the dependencies for the server to be installed. Steps are necessary for the server to run. 

Instead, receive errors from a number of packages failing to install. Console output is too verbose to get any sort of feedback of why this may be happening. 

#### Error Log

[log.txt](https://github.com/sufst/back-end/files/9847648/log.txt)
*Log file too big to look inside github*.

### Additional Notes
N/A

**Comment by EthanLatimerGB** [comment]
Deleting the Pipfile.lock file and installing dependencies completely fresh, attempting to install returns a `ResolutionFailure`.
```bash
(backend-venv) sufst@sufst:~/back-end$ pipenv install
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
Pipfile.lock not found, creating...
Locking [packages] dependencies...
Building requirements...
Resolving dependencies...
✘ Locking Failed! 

ERROR:pip.subprocessor:[present-rich] Getting requirements to build wheel exited with 1
[ResolutionFailure]:   File "/home/sufst/back-end/backend-venv/lib/python3.10/site-packages/pipenv/resolver.py", line 811, in _main
[ResolutionFailure]:       resolve_packages(
[ResolutionFailure]:   File "/home/sufst/back-end/backend-venv/lib/python3.10/site-packages/pipenv/resolver.py", line 759, in resolve_packages
[ResolutionFailure]:       results, resolver = resolve(
[ResolutionFailure]:   File "/home/sufst/back-end/backend-venv/lib/python3.10/site-packages/pipenv/resolver.py", line 738, in resolve
[ResolutionFailure]:       return resolve_deps(
[ResolutionFailure]:   File "/home/sufst/back-end/backend-venv/lib/python3.10/site-packages/pipenv/utils/resolver.py", line 1104, in resolve_deps
[ResolutionFailure]:       results, hashes, markers_lookup, resolver, skipped = actually_resolve_deps(
[ResolutionFailure]:   File "/home/sufst/back-end/backend-venv/lib/python3.10/site-packages/pipenv/utils/resolver.py", line 899, in actually_resolve_deps
[ResolutionFailure]:       resolver.resolve()
[ResolutionFailure]:   File "/home/sufst/back-end/backend-venv/lib/python3.10/site-packages/pipenv/utils/resolver.py", line 687, in resolve
[ResolutionFailure]:       raise ResolutionFailure(message=str(e))
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  You can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: Getting requirements to build wheel exited with 1
```
Entering `python --version` tells me that Ubuntu still uses the more latest version `3.10.*`, but when I uninstalled and rebooted to get it to use version 3.7, I may have broke my networking config somehow. 
Anyways, running `pipenv graph` returns:
![image](https://user-images.githubusercontent.com/62079750/197422678-422c439b-91bb-463a-a734-258f0b4e9999.png)

**Comment by AndreasDemenagas** [comment]
Solved with Docker implementation in #41
